### PR TITLE
Fixed coping and moving files to the same folder

### LINKED ISSUE #1242: No replace pop up when moving or coping files to the same folder
**Actual Behaviour**

When moving or coping the files to same folder from where they are selected
1. The files disappear(only in case of moving and not in case of coping) and when we open the  same folder, they appear now. ( see folder "stickers_s" in the gif below)
2. And when we copy the files to the same folder , there is no pop up asking if to replace those files or make more copies of it.

**Expected Behaviour**

There should be a pop up, asking if to replace those selected files as what happens normally in an operating system.

**Steps to reproduce it**

1. open the folder from which you want to move the files.
2. select those files.
3. select the folder in which you want to move or copy those files.
4. If you want to move or copy the files to the same folder there should be a pop up asking if to replace those files or make more copies of it.

**Screenshots of the issue**

mygif

**Would you like to work on the issue?**
Yes , I would like to work on this issue.

### PR DESCRIPTION
**Pull request by raj10071997**
Fix #1242  
Changes: Added a pop up when want to move the files to the same folder. And also corrected the UI update after the "move to" or " copy to" option is selected for the same folder from which files were selected from.

Screenshots for the change: 
screenshot

### CODE REVIEW

**Comment by raj10071997** [comment]
@mohitmanuja conflicts are resolved in the commit 191a393. Please review it.

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 1]
```diff
@@ -1720,7 +1723,45 @@ public void folderSelected(String path) {
...
+                            alertDialog.setPositiveButton( "Cancel", new DialogInterface.OnClickListener() {
+
+                                public void onClick(DialogInterface dialog, int id) {
+                                    dialog.cancel();
+                                } });
+
+                            alertDialog.setNegativeButton("Replace", new DialogInterface.OnClickListener() {
+
+                                public void onClick(DialogInterface dialog, int id) {
+                                    //mediaAdapter.swapDataSet(getAlbum().getMedia());
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Remove this line

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 2]
```diff
@@ -2166,4 +2188,53 @@ protected void onPostExecute(Void aVoid) {
             new PrepareAlbumTask().execute();
         }
     }
+
+
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Avoid unnecessary new lines

**Comment by raj10071997** [comment]
@anantprsd5 changes has been done. Please review it again.

**Pull request merged**